subprojects/android-test/rx-idler/build.gradle.kts

```diff
@@ -0,0 +1,10 @@
+plugins {
+    id("com.android.library")
+    id("kotlin-android")
+    `maven-publish`
+    id("com.jfrog.bintray")
+}
+
+dependencies {
+    implementation(Dependencies.AndroidTest.rxIlder)
+}
```

Comment:
How about versioning this library according to rx? `rx3-idler`
`com.squareup.rx.idler` has different artifacts for each version of rx: https://mvnrepository.com/artifact/com.squareup.rx.idler
It helps clients to consume only needed version.

Reply:
Done

Reply:
I meant the library itself: `subprojects/android-test/rx3-idler`

Reply:
Done